apps/middleman/src/app/app/(takeover)/stake/components/OwnerAddressStep/index.tsx

```diff
@@ -35,7 +35,7 @@ export default function OwnerAddressStep({onClose, onOwnerAddressSelected, selec
     Promise.all(
       connectedIdentities!.map((address) => {
         return getBalance(address).then((balance) => {
-          return {address, balance: balance / 1e6}
+          return {address, balance,}
```

Comment:
Don't think the `,` is necessary.

```suggestion
          return {address, balance}
```